ci_framework/roles/cifmw_cephadm/tasks/cephadm_config_set.yml

```diff
@@ -47,6 +47,25 @@
     {{ cifmw_cephadm_container_ns }}/{{ cifmw_cephadm_container_image }}
   changed_when: false
 
+- name: Set haproxy container image in ceph configuration
+  when:
+    - cifmw_cephadm_haproxy_container_image is defined
+    - not (cifmw_cephadm_default_container | default(false) | bool)
+  become: true
+  ansible.builtin.command: |
+    {{ cifmw_cephadm_ceph_cli }} config set mgr mgr/cephadm/container_image_haproxy \
+    {{ cifmw_cephadm_haproxy_container_image }}
+  changed_when: false
+
+- name: Set keepalived container image in ceph configuration
+  when:
+    - cifmw_cephadm_keepalived_container_image is defined
+    - not (cifmw_cephadm_default_container | default(false) | bool)
+  ansible.builtin.command: |
+    {{ cifmw_cephadm_ceph_cli }} config set mgr mgr/cephadm/container_image_keepalived \
```

Comment:
I think this task needs a `become: true`, @fyanac tried this downstream and was seeing permission issues accessing the spec yaml '/home/ceph-admin/specs/ceph_spec.yaml' when creating the container.

Reply:
Yes, I see `become: true` for similar tasks. It's just missing for this one.

Reply:
Nice catch! I splitted the tasks but i forgot to add `become: true` to the second one.

Reply:
> I think this task needs a `become: true`, @fyanac tried this downstream and was seeing permission issues accessing the spec yaml '/home/ceph-admin/specs/ceph_spec.yaml' when creating the container.

yes, it works. Thank you :)